txtl-liposome_thin-film.md

```diff
@@ -0,0 +1,109 @@
+
+B. Glass syringes of various sizes (ex. Hamilton gastight cat #:
+   14-815-238, but any will do) for use with lipids and chloroform
+
+C. Phosphate buffered saline 290 mOsm (Sigma, P4417-100TAB)
+
+D. Mini Extruder (Avanti Polar Lipids) 
+
+![Materials for TX-TL liposome protocol](txtl-liposome_thin-film-materials.png)
+
+## Method:
+
+Final reaction conditions:
+
+| Reagent | Source | Final working concentration |
+| ------- | ------ | --------------------------- |
+| DOPC    | Avanti 850375C | 2.5 E -5 mols/vial (1 mL of hydration yields 20 mM vesicles)* |
+| Rhodamine Lissamine PE | Avanti 810150 C | 0.1 mol % of POPC |
+| Cholesterol | Santa Cruz Biotech sc-202539 | 2X molarity of DOPC, if used |
+
+\* For microscopy, lower concentration of vesicles hydrated in low salt solutions yields higher quality vesicles for imaging (recommend you make 125 uM vesicles once hydrated)
+
+### Step 1: Prepare DOPC films (2.5E-4 mols DOPC/vial)
+
+1. Determine the volume and molarity DOPC vesicles you want to create
+   ex: 1 mL of 20 mM DOPC vesicles = 2.5E-5 mols of DOPC. The stock of
+   DOPC is 25 mg/mL. To make giant, nice looking vesicles, it’s best
+   to prepare a more dilute sample of vesicles (like 200 uM). For
+   small unilamellar vesicles, you can work with much higher
+   concentrations of lipids (we can go up to 130 mM for DOPC)
+
+2. Determine the dye and molar % of the dye : DOPC you would like
+   ex. for 0.1 mol% rhodamine DHPE :
+
+3. 2.5e-6 moles DOPC×(0.1 moles Rhod Liss PE)/(100 moles
+   DOPC)×L/(moles Rhod in stock)=volume Rhod Liss PE stock to add
+
+4. Add 2e-5 mols DOPC and appropriate amount of Rhodamine PE. Transfer
+   with a Hamilton (glass) syringe into a glass vial since you are
+   working with organic solvents. Cap and vortex briefly to ensure the
+   two components are well mixed.
+
+5. Remove cap and allow the chloroform to evaporate away in the hood.
+
+6. Place the vial into a vacuum chamber for > 1 hrs or leave overnight
+   in order to remove all residual traces of solvent. *Removal of
+   residual chloroform from the lipid film is very important for high
+   quality vesicles
+
+7. Cap and store vials in freezer until ready to be used and hydrate
+   with 1 mL of aqueous hydration buffer when you are ready to prepare
+   vesicles.
+
+### Step 2: Hydrate films and extrude vesicles
+
+1. Add 1 mL of PBS to each vial, place in 60°C oven for 1 hr, then
+   vortex 10 seconds. Lipids should easily assemble. If clumps or
+   aggregates persist after several vortexing attempts, this is often
+   a sign something went wrong in vesicle assembly. The heating step
+   is not necessary and can be excluded, but helps when other membrane
+   components are involved like diblock copolymers.
+
+2. Extrude films at room temperature through 100 nm polycarbonate
+   membranes using a mini extruder and heating block described on the
```

Comment:
A very common mistake (for students) is not realizing when something goes wrong with the extrusion. In particular, when the membrane breaks. If the protocol is intended for new-comers probably some explanations are needed here. Otherwise it is okay like it is.

Reply:
@PaolaTorre or @aaronengelhart: Can one of you suggest a change to address @p-stano's comment? I can implement once I know what to change.

Reply:
I think that the best is to follow the protocol for the mini-extruder from Avanti: the descriptiom of the process of extrusion is pretty detailed and easy to follow:https://avantilipids.com/divisions/equipment-products, comprise three steps: Assembly of the mini-extruder, Extrusion and Cleaning of the extruder.

Reply:
We included some information about this (and I think something in the video) in section 3 step 7 of: https://www.jove.com/video/57324/preparation-purification-and-use-of-fatty-acid-containing-liposomes

This is generally a by feel thing and once you've felt an intact membrane and a broken one, you'll know the difference. Trying with just supports and no membrane is good practice for learning the feel.

Reply:
Addressed in commit 3162c4a.